Using resourceVersion on comparison (#555)
giantswarm/app-operator: CHANGELOG.md

## [Unreleased]

### Fixed

> - Using resourceVersion of configmap on comparison instead of listing option.

Reviewer: ```suggestion
- Use resourceVersion of configmap for comparison instead of listing option.
```
